docs/sessionManagement.md

```diff
@@ -0,0 +1,79 @@
+# Session Management
+
+When Sample App is loaded by the client it is initialized with the express authentication middleware, [passportjs](http://www.passportjs.org/). Once passport is initialized `passport.session` is invoked generating the session. Using [express-session](https://www.npmjs.com/package/express-session) a cookie is generated that that represents the session. The strategy Sample App is utilizing to authenticate is `GraphQLLocalStrategy` from the [graphql-passport](https://www.npmjs.com/package/graphql-passport) package. Authenticating the user allows for certain action to take place, for example creating a basket. After successful authentication the anonymous user ID is stored in the session that corresponds to the cookie that is generated by express-session.
```

Comment:
There are two `that`
BTW: passport is a node.js middleware, not express. Although it works seamlessly with express based web app.

Reply:
@zsardone Maybe the wording is confusing here. The authentication middleware is invoked through express.use

Reply:
@echessman ok. can you remove one of the `that`?

Reply:
Hello @zsardone, I will remove it once @pdisabito has finished with his edits. I don't want to cause merge conflicts for him.

Reply:
sure.